### ISSUE TITLE
[11.5.0] Error: error:0308010C:digital envelope routines::unsupported

### ISSUE THREAD

**Issue by kasperg** [chromaui/chromatic-cli#1000]
**Bug report**

[We currently use `chromaui/action@v11` for running Chromatic in GitHub Actions](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/develop/.github/workflows/chromatic.yml). After v11.5.0 was released and the version of the action was bumped automatically the action has started to fail with the error below in seemingly unrelated PRs. 

As far as I can tell no other elements in the workflow was changed.

Example of a failing PR: https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/653

After locking the action at 11.4.0 the error no longer occurs: https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/654

The error code is the same as in #720 which is about a year old.

```
node:internal/crypto/hash:68
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:68:19)
    at Object.createHash (node:crypto:138:10)
    at module.exports (/home/runner/work/dpl-design-system/dpl-design-system/node_modules/@storybook/builder-webpack4/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/home/runner/work/dpl-design-system/dpl-design-system/node_modules/@storybook/builder-webpack4/node_modules/webpack/lib/NormalModule.js:417:16)
    at handleParseError (/home/runner/work/dpl-design-system/dpl-design-system/node_modules/@storybook/builder-webpack4/node_modules/webpack/lib/NormalModule.js:471:10)
    at /home/runner/work/dpl-design-system/dpl-design-system/node_modules/@storybook/builder-webpack4/node_modules/webpack/lib/NormalModule.js:503:5
    at /home/runner/work/dpl-design-system/dpl-design-system/node_modules/@storybook/builder-webpack4/node_modules/webpack/lib/NormalModule.js:358:12
    at /home/runner/work/dpl-design-system/dpl-design-system/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/home/runner/work/dpl-design-system/dpl-design-system/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at /home/runner/work/dpl-design-system/dpl-design-system/node_modules/loader-runner/lib/LoaderRunner.js:205:4
    at /home/runner/work/dpl-design-system/dpl-design-system/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:85:15
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

**Comment by cgbl-90** [comment]
@kasperg, thanks for reporting this! 
Our engineers have identified the issue and put a fix in place. 
For GitHub Action, you can update your workflow with:
`​uses: chromaui/action-canary@v11.5.0`
For CLI, try upgrading to `chromatic@11.5.1--canary.999.9346756383.0`

**Comment by mcstover** [comment]
Thanks for reporting this and to the team for finding a fix. In the meantime, we pinned our action to `chromaui/action@v11.4.0` in order to work around.

**Comment by kasperg** [comment]
For the record: We are still seeing this error even for a newer version like 11.5.3 which was just suggested by Dependabot.

**Comment by ghengeveld** [comment]
:rocket: Issue was released in [`v11.5.4`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.5.4) :rocket:

**Comment by tevanoff** [comment]
Thanks for reporting! This should be fixed in `v11.5.4`. Please let us know if you continue to experience the issue after upgrading.
